src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -895,6 +923,7 @@ def run(self):
             raise
         finally:
             logging.info("Tiled acquisition ended")
+            logging.info(f"The actual time taken per tile for each action is {self._save_time}")
```

Comment:
That's probably only useful for debugging. We shouldn't keep this in the final version.